initfiles/componentfiles/configxml/mysqlserver.xsd

```diff
@@ -116,12 +116,19 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
-            <xs:attribute name="computer" type="computerType" use="required">
+            <xs:attribute name="computer" type="computerType" use="optional">
                 <xs:annotation>
                     <xs:appinfo>
                         <tooltip>The computer on which the MySQL database is deployed.</tooltip>
                         <autogenforwizard>1</autogenforwizard>
                     </xs:appinfo>
+                </xs:annotation> 
+            </xs:attribute>
+            <xs:attribute name="host" type="xs:string" use="optional">
```

Comment:
@gaojx have you tested this? You may also need to upgrade the xsl files (esp_service.xsl, esp_service_wslogging.xsl, ?) where this attribute should be used.